lib/datasource/remote/firebase/regions/firebase_database_regions_repository.dart

```diff
@@ -203,6 +203,12 @@ class FirebaseDatabaseRegionsRepository extends FirebaseDatabaseService {
             .toList());
   }
 
+  Stream<RegionEventModel> getEventRegion(String regionId) {
+    return regionEventCollection.where('id', isEqualTo: regionId).snapshots().asyncMap((QuerySnapshot event) {
+      return RegionEventModel.mapToRegionEventModel(event as QueryDocumentSnapshot<Map<String, dynamic>>);
+    });
+  }
+
```

Comment:
@gguijarro-c-chwy If you could verify why it didn't work, since I'm using the one that listens to all the events, I don't know if there's any gain from listening to a particular event, if it's not worth it I'll remove it. 
I await your recommendation.